lib/Makefile

```diff
@@ -8,6 +8,9 @@
 # You may select, at your option, one of the above-listed licenses.
 # ################################################################
 
+# Note: by default, the static library is built single-threaded and dynamic library is built
```

Comment:
I think it's worthwhile to indicate something similar
in `lib/README.md`.

Reply:
Yep! In the process of updating the readme now.